lua/telescope/actions/init.lua

```diff
@@ -235,6 +235,13 @@ actions.center = function(prompt_bufnr)
   vim.cmd ":normal! zz"
 end
 
+local function jump_to_line()
+  local line_no = state.get_global_key "line_no"
+  if line_no then
+    vim.cmd["normal!"](line_no .. "G")
```

Comment:
we can drop that function and all post calls, by just changing something here:

https://github.com/nvim-telescope/telescope.nvim/blob/master/lua/telescope/actions/set.lua#L172-L177

Like we can change `row` to the global key if the global key is not nil (we just have to make sure we nil it again, if we delete `:123`)